MyMovies/Models/Movie+Convenience.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Movie+Convenience.swift
+//  MyMovies
+//
+//  Created by Cora Jacobson on 8/15/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+extension Movie {
+    
+    var movieRepresentation: MovieRepresentation? {
+    guard let title = title else { return nil }
+    
+        return MovieRepresentation(identifier: identifier?.uuidString ?? "", title: title, hasWatched: hasWatched)
```

Comment:
I would probably unwrap the uuidString in the guard up above instead of nil coalescing as you wouldn't want to send a representation with an id of an empty string.